lms/envs/common.py

```diff
@@ -2174,6 +2174,12 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.UserRateThrottle',
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'user': '50/minute',
```

Comment:
I picked this number from [ecommerce settings](https://github.com/edx/ecommerce/blob/master/ecommerce/settings/base.py#L480). But we do need a number user and [service user throttle rates.](https://github.com/edx/edx-platform/pull/14340/files#diff-82bda515879c96ab0641bfcbe58631c3R2182). 
@mattdrayer can you suggest an appropriate number for these ?